src/site/content/en/mini-apps/other-mini-app-runtime-environments/index.md

```diff
@@ -0,0 +1,264 @@
+---
```

Comment:
`web.dev/mini-app-runtime-environments`? I would like them all to start with the same prefix.

Reply:
Done.